package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "thehapiexperiment",
+  "version": "1.0.0",
+  "description": "Getting started with Hapi",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node bootstrap.js"
+  },
```

Comment:
how do you run this project?

Reply:
node src/app.js

should "node bootstrap.js" be "node src/app.js" ?